dags/common/esat/models.py

```diff
@@ -0,0 +1,64 @@
+import pandas as pd
+import sqlalchemy.types
+from sqlalchemy import Column
+from sqlalchemy.dialects.postgresql import insert as pg_insert
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import Session
+
+from dags.common import db
+
+
+DB_SCHEMA = "esat"
+
+# NOTE: when upgrading to sqlalchemy 2.0 or higher, we'll need to use the class DeclarativeBase
+EsatBase = declarative_base()
+
+
+def create_tables(variables):
+    db.create_schema(DB_SCHEMA)
+    build_esat_model(variables)
+    EsatBase.metadata.create_all(db.connection_engine())
+
+
+def build_esat_model(variables):
+
+    var_types = {}
+
+    for variable, variable_type_str in variables.items():
+        variable_type_str = variables[variable]["type"]
+        variable_type = getattr(sqlalchemy.types, variable_type_str, sqlalchemy.types.String)
+        if variable == "submission_id":
+            var_types[variable] = Column(variable_type, primary_key=True)
+        else:
+            var_types[variable] = Column(variable_type)
+
+    class_dict = {
+        "__tablename__": "questionnaire_2025",
+        "__table_args__": {"schema": DB_SCHEMA},
+        "__repr__": lambda self: "<Answers(submissionID=self.submissionID)>",
```

Comment:
```suggestion
        "__repr__": lambda self: f"<Answers(submissionID=self.submissionID)>",
```
Sinon ça va toujours afficher textuellement `<Answers(submissionID={self.submissionID})>` ;).